contracts/veLON.sol

```diff
@@ -238,6 +238,8 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
         _updateLockedPoint(_tokenId, _oldLocked, _lockedBalance);
 
         if (_value != 0 && _depositType != DepositType.MERGE_TYPE) {
+            tokenSupply = tokenSupply.add(_value);
```

Comment:
Use `prevSupply` to avoid 2nd storage read.

Reply:
replaced in [196b3c1](https://github.com/consenlabs/tokenlon-contracts/pull/75/commits/196b3c15fb149b918171fda65eefdfeb76da22bd)